doc/source/serve/getting_started.md

```diff
@@ -65,339 +71,279 @@ PyTorch, and Tensorflow for more info and examples:
-particular, the `summarize` function also has `min_length` and
-`max_length` parameters. Although we could expose these options as additional
-parameters in URL, Ray Serve also allows us to add more route options to the
-URL itself and handle each route separately.
+We compose our graph in line 50:
 
-Because this logic can get complex, Serve integrates with
-[FastAPI](https://fastapi.tiangolo.com/). This allows us to define a Serve
-deployment by adding the `@serve.ingress` decorator to a FastAPI app. For
-more info about FastAPI with Serve, please see {ref}`serve-fastapi-http`.
+```python
+deployment_graph = Summarizer.bind(Translator.bind())
+```
 
-As an example of FastAPI, here's a modified version of our `Summarizer` class
-with route options to request a minimum or maximum length of ten words in the
-summaries:
+Here, we bind `Translator` to its (empty) constructor arguments, and then we pass in the bound `Translator` as the constructor argument for the `Summarizer`. We can run this deployment graph using the `serve run` CLI command. Make sure to run this command from a directory containing a local copy of the `graph.py` code:
 
-```{literalinclude} ../../../python/ray/serve/examples/doc/e2e_fastapi_deployment.py
-:end-before: __fastapi_end__
+```console
+$ serve run graph:deployment_graph
+```
+
+We can use this client script to make requests to the graph:
+
+```{literalinclude} ../serve/doc_code/getting_started/model_graph.py
+:start-after: __start_client__
+:end-before: __end_client__
 :language: python
-:linenos: true
-:start-after: __fastapi_start__
 ```
 
-The class now exposes three routes:
+While the graph is running, we can open a separate terminal window and run the client script:
 
-- `/Summarizer`: As before, this route takes in article text and returns
-  a summary.
-- `/Summarizer/min10`: This route takes in article text and returns a
-  summary with at least 10 words.
-- `/Summarizer/max10`: This route takes in article text and returns a
-  summary with at most 10 words.
+```console
+$ python graph_client.py
 
-Notice that `Summarizer`'s methods no longer take in a Starlette `request`
-object. Instead, they take in the URL's `txt` parameter directly with FastAPI's
-[query parameter](https://fastapi.tiangolo.com/tutorial/query-params/)
-feature.
+c'était le meilleur des temps, c'était le pire des temps .
+```
 
-Since we still deploy our model locally, the full URL still uses the
-localhost IP. This means each of our three routes comes after the
-`http://127.0.0.1:8000` IP and port address. As an example, we can make
-requests to the `max10` route using this client script:
+Deployment graphs are useful since they let you deploy each part of your Machine Learning pipeline, such as inference and business logic steps, in separate deployments. Each of these deployments can be individually configured and scaled, ensuring you get maximal performance from your resources.
```

Comment:
```suggestion
Deployment graphs are useful since they let you deploy each part of your machine learning pipeline, such as inference and business logic steps, in separate deployments. Each of these deployments can be individually configured and scaled, ensuring you get maximal performance from your resources.
```